src/components/plugins/plugins-menu-overlay/plugins-menu-overlay.scss

```diff
@@ -0,0 +1,134 @@
+@import '~@playkit-js/playkit-js-ui';
+
+:global(.playkit-mini-audio-player-overlay) {
+  border-radius: 8px;
+
+  .plugins-menu-overlay {
+    background-size: cover;
+    background-position: center;
+    width: 100%;
+    height: 100%;
+
+    .plugins-menu-overlay-upper-layer {
+      width: 100%;
+      height: 100%;
+      color: $tone-1-color;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      border-radius: inherit;
+      font-size: 18px;
+
+      &.medium-small {
+        background-color: $paper-color;
+      }
+
+      &.large {
+        backdrop-filter: blur(16px);
+        background-color: rgba(0, 0, 0, 0.4);
+        box-shadow: inset 0 0 20px 0 #414141;
+      }
+
+      .menu {
+        margin: 10px;
+        width: 240px;
+      }
+    }
+  }
+}
+
+:global(.playkit-player.playkit-size-md),
```

Comment:
does this override it for the current player only or for all players on the page ?

Reply:
fixed